File: .NET/shikii.Hub.Core/Networking/Services/CTCMessage.cs
using shikii.Hub.DI;
using System;
using System.Collections.Generic;
using System.Text;


namespace shikii.Hub.Networking
{
    public class CTCMessage
    {
        public string AssemblyPath { get; set; }
        public String AssemblyName { get; set; }

        public String ClassName { get; set; }

        public string MethodName { get; set; }
        public Object[] Params { get; set; }

        public Object ReturnedData { get; set; }
        public String AliasDiName { get; set; } = null;
        public shikii.Hub.DI.DIClassAttribute.LifeCycleModes LifeCyleTimeMode { get; set; } = DIClassAttribute.LifeCycleModes.Singleton;
        public String ErrorMsg { get; set; }

        public void Clean()
        {
            AssemblyPath = null;
            AssemblyName = null;

            ClassName = null;

            MethodName = null;
            Params = null;

            ReturnedData = null;
            AliasDiName = null;
            ErrorMsg = null;
        }

    }
}

File: Java/app/src/main/java/System/Networking/Services/CTCMessage.java
package System.Networking.Services;

public class CTCMessage
{
    public String AssemblyPath ;
    public String AssemblyName;
    /**
     * 注意是 包名 + 类名
     */
    public String ClassName;
    public String MethodName ;
    public Object[] Params ;
    public Object   ReturnedData  ;
    public String AliasDiName = null;
    public  LifeCycleModes LifeCyleTimeMode  =  LifeCycleModes.Singleton;
    public String ErrorMsg ;

}
